model/src/main/java/org/cbioportal/genome_nexus/model/my_variant_info_model/Ann.java

```diff
@@ -0,0 +1,117 @@
+package org.cbioportal.genome_nexus.model.my_variant_info_model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class Ann
+{
+
+	@Field(value = "distance_to_feature")
+	public String distanceToFeature;
```

Comment:
Can you move all the fields to the beginning of the class and make them `private`, not `public`? In other words gather all (`private`) class members on the top, setters and getters on the bottom of the class. It is easier to read the classes that way.